### DISCUSSION TITLE
How Can I use onnxruntime_server work on CUDA backend.

### DISCUSSION THREAD

**Discussion by TaylorHere** [microsoft/onnxruntime#7053] [Other Q&A]
I build the onnxruntime_server binary from master branch  within a container build from [Dockerfile.ubuntu_server](https://github.com/microsoft/onnxruntime/blob/master/tools/ci_build/github/linux/docker/Dockerfile.ubuntu_server)
follow by [Server build Instructions](https://github.com/microsoft/onnxruntime/blob/master/docs/Server.md)
then I deploy the binary as
~~~Dockerfile
FROM nvidia/cuda:10.1-cudnn7-runtime-ubuntu16.04

RUN echo " " > /etc/apt/sources.list && \
    echo "deb http://mirrors.aliyun.com/debian buster main" >> /etc/apt/sources.list && \
    echo "deb http://mirrors.aliyun.com/debian buster-updates main" >> /etc/apt/sources.list && \
    apt-get update && apt-get install -y --allow-unauthenticated\
        software-properties-common \
        python3-pip \
        htop

RUN pip3 install --upgrade pip
RUN pip3 install onnxruntime-gpu==1.4.0

ADD onnxruntime_server /usr/local/bin/
ADD libonnxruntime.so.1.7.0 /usr/lib/libonnxruntime.so.1.7.0
ADD libre2.so.1v5 /usr/lib/x86_64-linux-gnu/libre2.so.1v5
ADD libcares.so.2 /usr/local/lib/libcares.so.2
~~~

But I found that there is no GPU memory or GPU-Util usage at all, looks like it's not working with a CUDA environment

Also I tried to use  -DUSE_CUDA=1 flag to build the binary, but I got output like:

~~~shell
CMake Warning:
  Manually-specified variables were not used by the project:

    USE_CUDA
~~~

How can I enable the CUDA support?

**Comment by TaylorHere** [comment]
Looks like I built version 1.7.0 and use a lower CUDA version, I'm trying to use the document referred version of CUDA and cuDNN

**Reply by TaylorHere** [reply]
it's not working, I still got this from server starting 
~~~shell
[2021-03-18 13:11:34.910] [ServerApp] [info] [ onnxruntime inference_session.cc:1132 Initialize]: Adding default CPU execution provider.
~~~

**Comment by pranavsharma** [comment]
onnxruntime server is deprecated. We don't support it any more.
